polaris/polaris/management/commands/poll_pending_deposits.py

```diff
@@ -203,7 +212,10 @@ def execute_deposits(cls):
                 ):
                     transaction.status = Transaction.STATUS.pending_trust
                     transaction.save()
-                continue
+                if not transaction.claimable_balance_supported:
```

Comment:
If `transaction.claimable_balance_supported` is `True`, we don't want to mark the transaction as `pending_trust`. `pending_trust` tells the wallet "the anchor is blocked". But Polaris isn't blocked if the wallet supports CBs.

Reply:
I think you need to check for `claimable_balance_supported` prior to marking the transaction as `pending_trust`.